src/components/viewContainer.js

```diff
@@ -236,7 +236,7 @@ import Dashboard from '../utils/dashboard';
     const mainAnimatedPages = document.querySelector('.mainAnimatedPages');
     let allPages = [];
     let currentUrls = [];
-    const pageContainerCount = 3;
+    const pageContainerCount = 1;
```

Comment:
This should not be necessary should it?

Reply:
the reason I did it b/c of the header tabs. for instance when you change for page to page the tab disappear. 

https://user-images.githubusercontent.com/32230989/165391872-40e08e81-c107-41df-bb04-45fac85b156b.mp4

Reply:
@thornbill I am thinking moving **header tabs** from **skin header** class to the **page view**?

Reply:
@thornbill updated to react router https://github.com/jellyfin/jellyfin-web/pull/3567/commits/4699e9de608096c889e16f7b7b510d7147e6f746